### ISSUE TITLE
MS5: 4/5 Stars

### ISSUE THREAD

**Issue by Skai-N** [edwinie/CDS-Onboarding-Neural-Networks#5]
Hey Neural Networks,

I really like how you added the questions you are investigating to your modeling page! The page looks pretty professional.

Here are some suggestions to get everything polished for presentations:

- Clean up the repo to move all assets (images and other data) for the website into a dedicated folder
- Would like to know which features you removed to create your reduced feature sets. Also it seems like the MSE was still very similar, so I think it'd be nice if you could talk about the tradeoffs between the size of the reduced feature set and the increase in MSE (is it worth increasing MSE if it reduced the feature set? What balance works best for your project?)
- I think it'd be beneficial to expand more on how an SVR differs from an SVM. Same goes for decision trees regressor and decision tree for classification
- Add an explanation for what the results/conclusions you can draw from each of your graphs to add more depth to the general answers
- Add a conclusion to discuss which model performed the best and why. Also interested to hear about what improvements you could've made to your process of data manipulation, feature engineering, or hyperparameter tuning to improve your results

Suggestions for making the website more presentable:

- Combine app and project introduction pages (potentially combine with data manipulation and continuous integration)
- Add more data visualizations
- More the sidebar labels in more a title format/style

Overall great job with this milestone!
